amazonia-1/amazonia-1-tiler-example.ipynb

```diff
@@ -0,0 +1,721 @@
+{
```

Comment:
<p>I wonder if we should drop the BBOX from this, so it shows all the tiles for the date range.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/asdi-examples/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>After doing this, I feel like it doesn't make sense in the context of the rest of the notebook. The whole premise is that we're interested in a specific area. How do I explain starting with the date range? Do I just reference the other notebook?</p>

Reply:
Update the text premise, it was duplicated from the STAC example. Sounds like the focus of this example is a tiled mosaic, so a date range not an AOI makes sense.

Reply:
Ok I'll also delete all the bbox stuff from the rest of the notebook then.